content/en/istio-ca-certs-integrations/gcp-cas-integration/_index.md

```diff
@@ -12,44 +12,74 @@ This approach enables the same root of trust for the root CA's workloads in GCP
+
+1. From the navigation menu, select Security → Certificate Authority Service.
+1. Click the **Create CA** button.
+1. Configure the CA type:
+    1. Select **Root CA**.
+    1. Select **365 days** for validity.
+    1. Select the **Enterprise** tier**.
+    1. Select the CAS's location from the **Location** list (e.g. `us-east1`).
+    1. Click **Next**.
+1. Configure the CA subject name (you can use your values here):
+    1. For **Organization (O)**, enter **Istio**.
+    1. For **Organization unit (OU)**, enter **engineering**.
+    1. For **Country name (C)**, enter **US**.
+    1. For **Locality name**, enter **Sunnyvale**.
+    1. For **CA Common name (CN)**, enter **getistio.example.io**.
+    1. For **Resource ID**, enter **getistio-example-io**.
+    1. Click **Next**.
+1. Configure the CA key size and algorithm:
+    1. Select **RSA PKCS1 2048 (SHA 256)**.
+    1. Click **Next**.
+1. Click the **Create** button to create the CAS.
+
+The figure below shows the summary page. Note that your page might look different if you configured your own CA subject name.
+
+![CAS Summary Page](./cas-summary.png)
 
-We can supply parameters related to connecting to GCP CAS and CSR creation through a YAML config file or command-line options. Creating a config file is recommended.
+### Configure GCP credentials
 
-Here's an example of a YAML config file with explained parameters.
+Ensure you have GCP credentials set up (e.g.`GOOGLE_APPLICATION_CREDENTIALS` environment variable has to point to the credentials) on a machine you're accessing the Kubernetes cluster from. Alternatively, if you installed GetIstio on Google Cloud Shell, the credentials are already set up.
+</details>
 
+## Creating CAS configuration
+
+We will use a YAML configuration to configure CAS and CSR creation. Use the YAML below as a template, and enter the CAS information from the CAS summary page on GCP:
+ 
 ```yaml
 providerName: "gcp"
 providerConfig:
   gcp:
-    #This will hold the full CA name for the certificate authority you created on GCP
-    casCAName: "projects/{project-id}/locations/{location}/certificateAuthorities/{YourCA}"
+    # This will hold the full CA name for the certificate authority you created on GCP
+    casCAName: "projects/tetrate-io-istio/locations/us-west1/certificateAuthorities/getistio-example-com"
 
 certificateParameters:
   secretOptions:
-    # Namespace where 'cacerts' Kubernetes secret is created on your target cluster
-    istioCANamespace: "istio-system"
-    # SecretFilePath is the file path used to store the Kubernetes Secret in yaml format
-    secretFilePath:
-    # Force flag when enabled forcefully deletes the `cacerts` secret
-    # in istioNamespace, and creates a new one.
-    force: true
+    istioCANamespace: "istio-system" # namespace where `cacerts` secrets live
+    force: true # force delete the `cacerts` secret and replace it with this new one
```

Comment:
Just a note - need to change this flag if the other PR in getistio is released to bintray.

Reply:
Yep, once that changes, I'll update it here as well (I think there might be a separate issue tracking it).